spec/models/ems_refresh/refreshers/openshift_refresher_spec.rb

```diff
@@ -101,10 +101,9 @@ def assert_specific_container_project
   def assert_specific_container_route
     @container_route = ContainerRoute.find_by_name("route-edge")
     @container_route.should have_attributes(
-      :ems_ref      => "f3b59d42-2870-11e5-b5bb-727174f8ab71",
-      :name         => "route-edge",
-      :host_name    => "www.example.com",
-      :service_name => "frontend",
+      :ems_ref   => "f3b59d42-2870-11e5-b5bb-727174f8ab71",
+      :name      => "route-edge",
+      :host_name => "www.example.com"
     )
```

Comment:
In the previous PR I asked you to add another check:

    @container_route.container_service.name.should be == "frontend"